Assets/Scripts/MainMenuManager.cs

```diff
@@ -0,0 +1,80 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.SceneManagement;
+
+public class MainMenuManager : MonoBehaviour {
+
+    private string[] menuItems = {"Start", "Exit"};
```

Comment:
Exit doesn't make sense on the web. We're making an About page soon, so change to that.